lib/tasks/create_tags.rake

```diff
@@ -570,6 +571,569 @@ namespace :tags do
       puts '-> PiMmS_TruSeq_adapters_v1 tags successfully created'
     end
 
+    desc 'Create SMRTbell Barcoded Adapter Plates ABCD tags'
+    task SMRTbell_Barcoded_Adapter_Plates_ABCD: :environment do
+      puts '-> Creating SMRTbell_Barcoded_Adapter_Plates_ABCD tag set and tags'
+      set = TagSet.pacbio_pipeline
+                  .find_or_create_by!(name: 'SMRTbell_Barcoded_Adapter_Plates_ABCD', uuid: '6ed141f6-4293-d3ba-933e-0b9b60c8a7ce')
```

Comment:
Is the UUID the same as SMRTLink prod or does it not matter for this set.

Reply:
This was the uuid provided by Craig so I presume yes.